src/client/eppo-client.spec.ts

```diff
@@ -317,43 +317,59 @@ describe('EppoClient E2E test', () => {
     });
   });
 
-  describe('UFC Shared Test Cases', () => {
+  describe.each(['Not Obfuscated', 'Obfuscated'])('UFC Shared Test Cases %s', (obfuscationType) => {
```

Comment:
Using dataproviders allows reducing our setup and test case running code duplication